docs/PacketFence_Clustering_Guide.asciidoc

```diff
@@ -9,770 +9,85 @@ include::includes/global-attributes.asciidoc[]
 
-Restore previous MariaDB data
+  mysql -u root -p
 
-  # mv /var/lib/mysql.bak/* /var/lib/mysql/ #Optional
+    mysql> CREATE USER 'pfcluster'@'%' IDENTIFIED BY 'aMuchMoreSecurePassword';
+    mysql> GRANT RELOAD, LOCK TABLES, REPLICATION CLIENT ON *.* TO 'pfcluster'@'%';
 
-Start MariaDB
+    mysql> CREATE USER 'pfcluster'@'localhost' IDENTIFIED BY 'aMuchMoreSecurePassword';
+    mysql> GRANT RELOAD, LOCK TABLES, REPLICATION CLIENT ON *.* TO 'pfcluster'@'localhost';
 
-  # systemctl start mariadb
+    mysql> FLUSH PRIVILEGES;
 
-Execute the secure installation script in order to set the root password, remove the test databases and anonymous user created by default:
+Next, you need to remove the empty users from your MySQL database:
 
-  # /usr/bin/mysql_secure_installation
-
-Make sure to stop MariaDB:
-
-  # systemctl stop mariadb
-
-MariaDB network configuration
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-In order for your PacketFence cluster to communicate properly with the MariaDB database, you need to have MariaDB bind on the management IP address.
-
-Adjust your MariaDB configuration in `/etc/my.cnf` on both servers and make sure the bind_address line is :
-
-  bind_address=<management_ip_address>
+  # mysql -u root -p
+    mysql> delete from mysql.user where user = '' ;
+    mysql> flush privileges;
 
 Step 2 : Server configuration
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-You need to make sure the interfaces name will be the same on both servers. See 'Setting the interfaces name on CentOS 7' in the Appendix section of this document.
+First, you need to make sure the interfaces name will be the same on both servers. See 'Setting the interfaces name on CentOS 7' in the Appendix section of this document if all your servers don't already have the same interface names.
 
-You will need to configure the server so the services can bind on IP addresses they don't currently have configured. This allows faster failover of the services.
+Next, you will need to configure the server so the services can bind on IP addresses they don't currently have configured. This allows faster failover of the services.
 
-On CentOS, add the following line in /etc/sysctl.conf and then reload with 'sysctl -p'
+On all your servers, add the following line in `/etc/sysctl.conf` and then reload with 'sysctl -p'
 
   net.ipv4.ip_nonlocal_bind = 1
 
-Create the PEM that combines the key and certificate for the http services. Adapt to your own paths if you are using different certificates.
+Now, on the first server, create the PEM that combines the key and certificate for the http services. Adapt to your own paths if you are using different certificates. Most important is that the destination PEM file is `/usr/local/pf/conf/ssl/server.pem` as its the one used in haproxy.
```

Comment:
I suggest something like "What matters is that the path to the PEM file should be `/usr/local/pf/conf/ssl/server.pem` since HAProxy will expect it to be there.